README.md

```diff
@@ -74,7 +74,7 @@ When using a YAML file containing queries, you can specify the following paramet
 ## Compatibility
```

Comment:
perhaps removing this and pointing to the docs, so we have a single source of truth WDYT?